### DISCUSSION TITLE
Quais são as fontes para ficarem atualizados ?

### DISCUSSION THREAD

**Discussion by mksanti** [frontendbr/forum#33] [Discussão]
Vendo as discussões que surgiram esta semana aqui no fórum, vi que a galera disponibiliza muitos links sobre diversas coisas, então que tal juntar tudo aqui mostrando os links/sites/repos que vocês utilizam para ficarem atualizados.

**Comment by raphaklaus** [comment]
Bom, eu não tenho um feed especifico... Heheh, vou explicar como faço:
- Sigo grupos de desenvolvimento no Facebook (não só brasileiros. Por exemplo o NodeJS Latino é muito forte). Isso faz com que, querendo ou não, eu esbarre com noticias quase que sempre.
- Sigo pessoas chaves aqui no Github, então na minha home aparece sempre as atividades destas pessoas, seja em projetos delas, stars que deram.. E direto vejo coisas novas e interessantes.
- Eventos. Tento participar do máximo de eventos possíveis que estão ao meu alcance. Lá você terá a oportunidade de ver pessoas apresentando updates bacanas.
- Eu seguia o [DailyJS](http://dailyjs.com/), mas ele acabou. :disappointed:
- Às vezes escuto alguns podcasts, tipo o [ZOFE](http://zofe.com.br/) e [GrokPodcast](http://www.grokpodcast.com/)

**Comment by vitorbritto** [comment]
Mandem referências para o [Cerebro](https://github.com/cerebrobr). =]

Fiz do Twitter um feed de notícias. Tenho listas criadas para assuntos específicos.

**Comment by willianjusten** [comment]
Eu sigo muitas pessoas do twitter, também leio blogs e assino algumas weeklys.
Eu fiz um post sobre várias coisas de frontend e em uma das coisas, eu passei alguns links, talvez interesse:
- [Como se tornar um Desenvolvedor Frontend](http://willianjusten.com.br/como-se-tornar-um-desenvolvedor-front-end/#update)

**Comment by willianjusten** [comment]
Me lembrei que tem também um site fantástico para isso:
- [How to keep up to date with frontend technologies ](https://uptodate.frontendrescue.org/)

**Comment by alisonmonteiro** [comment]
@willianjusten post muito bom, esse "Como se tornar um Desenvolvedor Frontend"! 
Me fez descobrir que to bem mais a frente do que imaginava.
Mas ainda tenho muito pra aprender! :+1:

**Comment by willianjusten** [comment]
Opa @alisonmonteiro , sempre se tem o que aprender! =)
Só não pode parar.

**Comment by vitor-mendrone-deel** [comment]
@mikejavier tenho um repo com os sites que leio diariamente: 

https://github.com/Mendrone/Feed-De-Leitura

Minha sugestão é adicionar todos ao [Feedly](feedly.com), assim você não precisa necessariamente acessá-los todo dia e mantém tudo organizado. :smile:

_edit_

A sugestão de seguir um pessoal no Twitter, do @willianjusten é bem válida também. O @LFeh, eu, o próprio @willianjusten e mais uma galera sempre estamos postando o que lemos de interessante por ai

**Comment by willianjusten** [comment]
Me lembrei de uma iniciativa bacana do @LFeh mesmo, que é o:
- [Lista de blogs e sites úteis para desenvolvedores Front-End - pt-BR](https://github.com/LFeh/feed-list)

Lá tem bastante coisa maneira em pt-BR =)

**Comment by rickbenetti** [comment]
Com exceção aqui do Fórum no Github e por ser designer também, tenho uma lista de sites estrangeiros que sigo e acompanho todos os dias (tendo uma média de 300 ~ 500 notícias lidas por dia rs).

Segue alguns feeds interesantes:
- http://feeds.feedburner.com/htmlitedit
- http://feeds2.feedburner.com/24thfloor
- http://rss1.smashingmagazine.com/feed/
- http://feeds.feedburner.com/bavotasan
- http://feeds2.feedburner.com/Paulundcouk
- http://speckyboy.com/feed
- http://feeds.feedburner.com/fubiz
- http://rss.html.it/webnews.xml
- http://tableless.com.br/feed/
- https://dribbble.com/

---

Lista de utilidade do meu Twitter:
- https://twitter.com/_RKB81/lists/wdsign
- https://twitter.com/frontendrescue/lists/cool-people

---

Minha comunidade no Google Plus:
- https://plus.google.com/communities/104431911254900556469
  (acabei que no último ano tenho sido um tanto relapso)

---

Blogs que acesso 1 vez por mês:
- http://smashinghub.com/
- http://www.onextrapixel.com/
- http://fr.tuto.com/
- https://www.christophh.net/

E últimamente tenho visto muita coisa bacana no Medium.com

**Comment by marquinhusgoncalves-zz** [comment]
Sou leitor compulsivo gosto de estar conectado e por dentro de tudo que acontece pena que o dia tenha apenas 24hs então segue:
- Vou em todos eventos possíveis;
- Tenho no Feedly os sites e blogs mais importantes da área uso plugin para o Chrome;
- Sigo pessoas ativas e da área no Twitter ❤️ [@marquinhusgonc](https://twitter.com/marquinhusgonc);
- Faço o mesmo no Github porém lá vejo o que as pessoas estão fazendo, iniciando projetos, "startando" e forkando;
- Participo de alguns grupos do Slack e do Facebook;

**Comment by felipefialho** [comment]
Perfeita sua rotina @marquinhusgoncalves, acho que é a melhor forma de se manter atualizado.

**Comment by robisson** [comment]
Eu particularmente, faço similar ao @marquinhusgoncalves  , procuro pegar referências que postam conteúdo de qualidade e concentrar elas no  twitter, assim posso ver tudo em único lugar. E atualmente procuro escutar os podcasts que encontro sobre desenvolvimento, que por enquanto em português são poucos, mas para a galeira que sabe inglês tem várias opções legais. E o podcast me permite escutar e trabalhar ao mesmo, mesmo que em alguns momentos eu nem esteja prestando atenção. rsrs

**Comment by caiorss** [comment]
Reddit:
- http://www.reddit.com/r/haskell
- http://www.reddit.com/r/python
- http://www.reddit.com/r/fsharp
- http://www.reddit.com/r/clojure
- http://www.reddit.com/r/scala
- http://www.reddit.com/r/linux
- http://www.reddit.com/r/programming
- http://www.reddit.com/r/ocaml
- http://www.reddit.com/r/javascript
- http://www.reddit.com/r/brdev

Existem subreddits para todas linguagens, para quem não conhece é um fóruns que agrega links, posts e discussões. No reddit o pessoal manda links de toda a Internet o que facilita a procura.

HackerNews Ycombinator 
- https://news.ycombinator.com/

Google Scholar/ Acadêmico - Para encontrar teorias que aumentem a produtividade.

Listas de Emails de diversas comunidades dedicadas a linguagem de programação no site Gmane: http://gmane.org
